src/pohjavirta/response.clj

```diff
@@ -97,6 +99,28 @@
         (.getResponseSender)
         (.send ^ByteBuffer body)))
 
+  InputStream
+  (send-body [stream exchange]
+    (let [bytes (byte-array 8192)
+          send-more (fn send-more [^Sender sender]
+                      (let [count (.read stream bytes 0 8192)]
```

Comment:
This is a blocking call. We should test if we are running in an XNIO pool and dispatch the `exchange` into a worker pool if we are. There is an example how to do this in https://github.com/metosin/pohjavirta/blob/master/src/pohjavirta/exchange.clj